db/channel_cache.go

```diff
@@ -187,6 +192,22 @@ func (c *channelCache) _pruneCache() {
 		c.logs = c.logs[pruned:]
 	}
 
+	return pruned
+}
+
+func (c *channelCache) pruneCacheAge() {
```

Comment:
Probably don't need separation of locking/non-locking versions, since locking pruneCacheLength is being removed.